packages/@aws-cdk/core/lib/deployment-configuration.ts

```diff
@@ -0,0 +1,635 @@
+   * The role to use to publish assets to this environment
+   *
+   * You must supply this if you have given a non-standard name to the publishing role.
+   *
+   * The placeholders `${Qualifier}`, `${AWS::AccountId}` and `${AWS::Region}` will
+   * be replaced with the values of qualifier and the stack's account and region,
+   * respectively.
+   *
+   * @default 'arn:${AWS::Partition}:iam::${AWS::AccountId}:role/cdk-bootstrap-publishing-role-${AWS::AccountId}-${AWS::Region}'
+   */
+  readonly assetPublishingRoleArn?: string;
+
+  /**
+   * External ID to use when assuming role for asset publishing
+   *
+   * @default - No external ID
+   */
+  readonly assetPublishingExternalId?: string;
+
+  /**
+   * The role to assume to initiate a deployment in this environment
+   *
+   * You must supply this if you have given a non-standard name to the publishing role.
+   *
+   * The placeholders `${Qualifier}`, `${AWS::AccountId}` and `${AWS::Region}` will
+   * be replaced with the values of qualifier and the stack's account and region,
+   * respectively.
+   *
+   * @default 'arn:${AWS::Partition}:iam::${AWS::AccountId}:role/cdk-bootstrap-deploy-action-role-${AWS::AccountId}-${AWS::Region}'
+   */
+  readonly deployActionRoleArn?: string;
+
+  /**
+   * The role CloudFormation will assume when deploying the Stack
+   *
+   * You must supply this if you have given a non-standard name to the execution role.
+   *
+   * The placeholders `${Qualifier}`, `${AWS::AccountId}` and `${AWS::Region}` will
+   * be replaced with the values of qualifier and the stack's account and region,
+   * respectively.
+   *
+   * @default 'arn:${AWS::Partition}:iam::${AWS::AccountId}:role/cdk-bootstrap-cfn-exec-role-${AWS::AccountId}-${AWS::Region}'
+   */
+  readonly cloudFormationExecutionRole?: string;
+
+  /**
+   * Qualifier to disambiguate multiple environments in the same account
+   *
+   * You can use this and leave the other naming properties empty if you have deployed
+   * the bootstrap environment with standard names but only differnet qualifiers.
+   *
+   * @default 'hnb659fds'
+   */
+  readonly qualifier?: string;
+}
+
+/**
+ * Uses conventionally named roles and reify asset storage locations
+ *
+ * This DeploymentConfiguration is the only DeploymentConfiguration that generates
+ * an asset manifest, and is required to deploy CDK applications using the
+ * `@aws-cdk/app-delivery` CI/CD library.
+ *
+ * Requires the environment to have been bootstrapped with Bootstrap Stack V2.
+ */
+export class DefaultDeploymentConfiguration implements IDeploymentConfiguration {
+  private stack!: Stack;
+  private bucketName!: string;
+  private repositoryName!: string;
+  private deployActionRoleArn!: string;
+  private cloudFormationExecutionRoleArn!: string;
+  private assetPublishingRoleArn!: string;
+
+  private readonly assets: asset_schema.ManifestFile = {
+    version: asset_schema.AssetManifestSchema.currentVersion(),
+    files: {},
+    dockerImages: {},
```

Comment:
```suggestion
    images: {},
```

?

Reply:
Why? They are docker images. The word "image" by itself does not immediately conjure up "docker" for me. I find it nicely disambiguates.